db/migrate/20230308160615_create_data_resources.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class CreateDataResources < ActiveRecord::Migration[7.0]
+  def change
+    create_table :data_resources, id: :uuid do |t|
+      t.text :identifier
+      t.text :title
+      t.text :description
+      t.text :keywords, array: true, default: []
+      t.text :theme
+      t.text :license
+      t.text :version
+      t.text :contact_name
+      t.text :contact_email
+      t.text :alternative_title
+      t.integer :access_rights
+      t.integer :security_classification
+      t.date :issued
```

Comment:
is there a reason `issued` is a date, not a datetime?

Reply:
https://docs.google.com/document/d/13KqG1Zom0YqCehPHagCnV6ADOwj8k6qcv7Us4UDWnNg/edit?pli=1#heading=h.7l22ptgj7j5m talks about a date of issue.
I also feel like a datetime is too specific for this kind of data, and fear we would end up with a date + 00:00:00

Reply:
ok that's fine